docs/source/nbclassic.rst

```diff
@@ -0,0 +1,84 @@
+.. _htmlnotebook:
+
+Jupyter NbClassic
+====================
+
+Introduction
+------------
+
+The nbclassic package is the implementation of the classic Jupyter
+Notebook-v6 as a Jupyter Server extension. As proposed in the accepted
+`Jupyter Enhancement Proposal #79`_ the development of this package
+is due to plans being carried out to create a Notebook-v7, that is based on
+the modern JupyterLab code-base. 
+
+While Notebook-v7 provides a user experience equivalent to that of the
+classic Notebook-v6, the technology stack used is incompatible with
+that which many Jupyter Notebook users have developed their Jupyter
+Notebook extensions with. Users may find themselves in need of a
+way to continue using the Jupyter Notebook-v6 tech-stack as they
+transition to using the Jupyter Notebook-v7. The NbClassic package
+intends to address that need.
+
+.. _Jupyter Enhancement Proposal #79: https://jupyter.org/enhancement-proposals/79-notebook-v7/notebook-v7.html
+
+
+NbClassic Usage
+---------------
+
+Installation
+~~~~~~~~~~~~
+
+Installing from PyPI:
+``> pip install nbclassic``
+This will automatically enable the extension in Jupyter Server.
+
+Launch directly:
+``> jupyter nbclassic``
+
+Alternatively, you can run Jupyter Server and visit the `/tree` endpoint:
+``> jupyter server``
+
+
+NbClassic in the Jupyter Ecosystem
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Maybe refer to https://github.com/jupyter/notebook-team-compass/issues/5#issuecomment-1085254000

Reply:
Okay awesome! Thank you for the link, I will add it to the documentation too!